panacea/web/static/js/app.js

```diff
@@ -46,12 +46,15 @@ async function handleDrugsResponse(response) {
       View.displayNoDrugsError();
     }
 
+    // fetching the pml here, TODO: decide when is best to do this
+    const pml = await Request.pml(ast);
```

Comment:
I've just realised there's a potential issue using `Request.pml(...)` here. The download button strips comments, so the line numbers may all be off.

This depends on how Yecc/Leex handle skipping tokens - I'll investigate a bit now. We may need to return the _exact_ PML they gave us.

We could alter Panacea to return the PML in the `/api/drugs` response or have JS read the file contents. Alternatively, we could change the parser to not strip comments - potentially messy.